steps/resources/steps.go

```diff
@@ -149,8 +151,10 @@ func (f *ResourcesFeatureContext) userHasUploadedAFileWithContentInTheHomeDirect
 		}
 	}
 
+	length := int64(len(content))
 	uReq := &providerv1beta1.InitiateFileUploadRequest{
-		Ref: resourceRef,
+		Opaque: utils.AppendPlainToOpaque(nil, net.HeaderUploadLength, strconv.FormatInt(length, 10)),
```

Comment:
um .... maybe just use `Upload-Length` as string ... instead of having to pull in ocis?
```suggestion
		Opaque: utils.AppendPlainToOpaque(nil, "Upload-Length", strconv.FormatInt(length, 10)),
```

Reply:
:+1: